### DISCUSSION TITLE
Css files not loading with actix

### DISCUSSION THREAD

**Discussion by DogsCodesStudio** [sunng87/handlebars-rust#527] [Q&A]
I have 

```
use actix_files::Files;
use actix_web::{get, web, App, HttpResponse, HttpServer};
use handlebars::Handlebars;
use serde_json::json;

#[get("/")]
async fn tests(hb: web::Data<Handlebars<'_>>) -> HttpResponse {
    let data = json!({
        "user": 25,
        "data": 12
    });
    let body = hb.render("404", &data).unwrap();
    HttpResponse::Ok().body(body)
}

#[actix_web::main]
async fn main() -> std::io::Result<()> {
    let mut handlebars = Handlebars::new();
    handlebars
        .register_templates_directory(".html", "/var/www/templates")
        .unwrap();

    // let tests = Files::new("/static", ".").show_files_listing();
    let handlebars_ref = web::Data::new(handlebars);

    HttpServer::new(move || App::new()
    .service(Files::new("/var/www/templates", ".").show_files_listing())
        .app_data(handlebars_ref.clone())
        .service(tests))
        .bind(("127.0.0.1", 25565))?
        .run()
        .await
}

async fn not_found(hb: web::Data<Handlebars<'_>>) -> HttpResponse {
    let data = json!({
        "user": 25,
        "data": 12
    });
    let body = hb.render("404", &data).unwrap();
    HttpResponse::Ok().body(body)
}``` and 

```
[dependencies]
actix-web = "4"
actix-files = "0.6.2"
handlebars = { version = "4.2.1", features = ["dir_source"] }
serde = { version = "1.0.143", features = ["derive"] }
serde_json = "1.0.72"
``` as dependencies 

with my template file being 
![image](https://user-images.githubusercontent.com/98062743/186021196-27a77d0b-7a1f-4b29-b918-bdc7306eb397.png)
but it doesn't seem to be loading the css file

**Comment by sunng87** [comment]
Typically template engine is not responsible for css loading. Could you please check your generated content to see if it has correct css file path?

**Comment by DogsCodesStudio** [comment]
![image](https://user-images.githubusercontent.com/98062743/186267029-143cf1cc-ae31-4431-9434-73faa92847bd.png)
had link ref as 404.css and that didn't work but that did kinda weird but alright

Edit: might be because im doing 
` .service(Files::new("/var/www/templates", ".").show_files_listing())` cause when i remove the style it still formats the same as if its there

**Comment by sunng87** [comment]
You are using the file system absolute path for the css file. Just remove the path `/var/www/templates/`  from html and see if it works.

**Comment by DogsCodesStudio** [comment]
The answer is no, im not sure what is going on for it not to work but its not working. I tried with `    <link href="/var/www/templates/404.css" rel="stylesheet">
    {# <link href="404.css" rel="stylesheet"> #}` Both of those commented out and uncommented out. None of it seems to work now don't think im doing anything different.
